README.md

```diff
@@ -71,8 +71,9 @@ User.all #Dolly::Collection #<#User...>, <#User...>
 user = User.find "a1b2d3e" #<#User...>
 user.name
 
-#Return a User object based on the custom view.
-user = User.view 'view_name', {key: ["a", "b", "c"], reduce: true}
+#Return a User object based on the custom view. `include_docs` is always true for this method.
```

Comment:
JSON String, that's why I added the `from_json` call, as it previously seemed to indicate you'd be getting a User instead of a String.